Changelog.md

```diff
@@ -1,3 +1,8 @@
+## 3.4.2 - 2022-02-17
+
+### Fixed
+- Loading larger files are now supported.
```

Comment:
Should I add that one should be careful of memory usage? We do not have a max limit. This means that if a user A is sent a file from another user B with larger memory than user A, then user B may experience significant performance issues.